### PR TITLE
run the convert.sh script by CDK2020 to replace namespaces to be comp…

### PR DESCRIPTION
**Pull request by AakLak**
Run this script:
https://discuss.flarum.org/d/6573-convert-your-beta-7-compatible-extension-to-work-on-beta-8

### CODE REVIEW

File: src/Listener/AddClientAssets.php [hunk 1]
```diff
@@ -21,13 +21,13 @@ class AddClientAssets
      */
     public function subscribe(Dispatcher $events)
     {
-        $events->listen(ConfigureWebApp::class, [$this, 'addAssets']);
+        $events->listen(Rendering::class, [$this, 'addAssets']);
     }
 
     /**
      * @param ConfigureClientView $event
```
Comment (tobyzerner) [inline src/Listener/AddClientAssets.php]: Please update this docblock

Comment (AakLak) [reply src/Listener/AddClientAssets.php]: Sorry, I'm not sure what it should be instead. New to Flarum.
I see this on the namespace changes thread:
```{Event/ConfigureWebApp → Frontend/Event/Rendering}```

Comment (tobyzerner) [reply src/Listener/AddClientAssets.php]: No worries! Simply change line 28:

```
      * @param ConfigureClientView $events
```

to:

```
      * @param Rendering $events
```

Comment (AakLak) [reply src/Listener/AddClientAssets.php]: Oh duh! Think I got it, thanks.

**Comment by tobyzerner** [comment]
Thanks @AakLak :D

**Comment by franzliedke** [comment]
Nooo... this should have gone to the `next-back` branch. ;)

**Comment by tobyzerner** [comment]
Oops 😳 Fixed, I think

**Pull request merged**